src/components/PollResultModal/PollResultChart/PollResultChartBar.test.tsx

```diff
@@ -152,4 +177,160 @@ describe('<PollResultChartBar/>', () => {
       },
     });
   });
+
+  it('should change the diagrams colors in dark mode', () => {
+    jest
+      .mocked(useThemeSelection)
+      .mockReturnValue({ theme: 'dark', setTheme: jest.fn(), isModal: false });
+    const pollResult: SelectPollResults = {
+      poll: mockPoll({
+        content: {
+          pollType: PollType.Open,
+          startTime: '2020-01-01T03:33:55Z',
+        },
+      }),
+      results: {
+        votes: {
+          '@user-1': '1',
+          '@user-2': PollInvalidAnswer,
+          '@user-3': '2',
+          '@user-4': '1',
+        },
+      },
+      votingRights: ['@user-1', '@user-2', '@user-3', '@user-4'],
+    };
+
+    render(<PollResultChartBar pollResults={pollResult} />, {
+      wrapper: Wrapper,
+    });
+
+    expect(screen.getByText('SimpleBarChart')).toBeInTheDocument();
+
+    expect(mockSimpleBarChart).toBeCalledWith({
+      data: [
+        { group: 'Yes', value: 2 },
+        { group: 'No', value: 1 },
+      ],
+      options: {
+        color: {
+          scale: {
+            No: '#CBB701',
+            Yes: '#8AB3FF',
+          },
+        },
+        axes: {
+          bottom: {
+            mapsTo: 'group',
+            scaleType: 'labels',
+          },
+          left: {
+            mapsTo: 'value',
+            ticks: {
+              values: [0, 1, 2],
+            },
+          },
+        },
+        height: '270px',
+        theme: 'g100',
+        toolbar: {
+          enabled: false,
+        },
+        tooltip: {
+          enabled: false,
+        },
+      },
+    });
+  });
+
+  it('should repeat the colors if the answers are more then 7', () => {
```

Comment:
I wouldn't test it via a component test. If you want to make sure that it works correctly, create a unit test for `createObjectColors`. They are faster and smaller and you could do more smaller tests.